pkg/storage/local_docker_server_stages_storage.go

```diff
@@ -189,7 +189,7 @@ func (storage *LocalDockerServerStagesStorage) GetManagedImages(ctx context.Cont
 
 func (storage *LocalDockerServerStagesStorage) GetStagesIDsByDigest(ctx context.Context, projectName, digest string) ([]image.StageID, error) {
 	filterSet := filters.NewArgs()
-	filterSet.Add("reference", fmt.Sprintf(LocalStage_ImageRepoFormat, projectName))
+	filterSet.Add("reference", projectName)
```

Comment:
We keep the formats in constants in order to see the full picture in one place and change them globally.

Reply:
Agree, was sure that I've left this unchanged, reverting.